partials/hero.hbs

```diff
@@ -2,30 +2,39 @@
 If the template in question has a background image, then we render responsive image styles
 for it, and apply those styles to the <header> tag.
 --}}
+{{!--
+If the template in question has a background image, then we render responsive image styles
+for it, and apply those styles to the <header> tag.
+--}}
 
 {{#if background}}
   {{#contentFor "herobackground"}}
-  <style>
-    .m-hero__picture {
-      background-image: url({{img_url background size='xl'}});
-    }
-  
-    @media(max-width: 1000px) {
+    <style>
       .m-hero__picture {
-        background-image: url({{img_url background size='l'}});
+        display: flex;
+        align-items: center;
+        justify-content: center;
       }
-    }
-  
-    @media(max-width: 600px) {
-      .m-hero__picture {
-        background-image: url({{img_url background size='m'}});
+      .m-hero__picture img{
+        width: 100%;
```

Comment:
You can also move these styles to the `src/sass/components/hero/_hero.scss` file. You can add them after [line 50](https://github.com/eddiesigner/liebling/blob/master/src/sass/components/hero/_hero.scss#L50) like so:

```sass
.m-hero__picture {
  /*
    Add the styles after
    &.in-post {
      opacity: 1;
    }
  */
  img {
    width: 100%;
    height: 100%;
    object-fit: cover;
  }
}
```

In fact, all these media queries are not longer neccessary with this new implementation ;)

Reply:
Pushing a fix to all these comments, I'm not sure I understand the last remark here. Is there some media query you'd like me to remove?

Reply:
Nope, that's fine 👍